pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -194,7 +194,7 @@ func (k *KubeData) initNodeInformer(informerFactory informers.SharedInformerFact
 		return &Info{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      node.Name,
-				Namespace: node.Namespace,
+				Namespace: "",
```

Comment:
Nitpick: just remove the line and Go by default assigns the empty string.
```suggestion
```

Reply:
I'm not 100% sure of that, when it comes to json serialization, empty values != zero-values , right?

Reply:
(I've been tricked by this oddity quite a few times in the past!)

Reply:
In Go, there is no difference between zero value and empty value. In the case of string, the empty string is the zero value.

Maybe you got this issue when serializing nil pointers vs pointers to empty struct.

Reply:
It's this kind of things:

```go
	var slice []string
	js, _ := json.Marshal(slice)
	fmt.Println(string(js)) // prints null

	slice = []string{}
	js, _ = json.Marshal(slice)
	fmt.Println(string(js)) // prints []
```

Maybe this is not about zero-value, but about golang being very permissive on nil slices, acting as if it was an empty slice most of the time.

But apparently you're right, for string there's no such issue